components/user-avatar/src/user-avatar.stories.js

```diff
@@ -0,0 +1,26 @@
+import { Box } from '@dhis2-ui/box'
+import React from 'react'
+import { UserAvatar } from './user-avatar.js'
+
+const description = `
```

Comment:
Defining a description in this way is now deprecated, and should instead be done using a `/**` comment above the component's definition. This allows the API docs to render the description too. We'll also want a link to the design system.

```js
// In components/user-avatar/src/user-avatar.js:

/**
An avatar is a visual icon that represents a user.

Avatars are used to give extra information when a user is mentioned or displayed in DHIS2. The avatar contains either user uploaded photograph, initials or a default icon. The avatar is intended to give context to a user and help to quickly identify different users. An avatar is usually shown alongside the user name, but can be used alone to show a visual hint of a user.

See specification: [Design System](https://github.com/dhis2/design-system/blob/master/atoms/avatar.md)
*/

export const UserAvatar = ({ avatarId, name, dataTest }) => {
```